models/staging/src_servicenow.yml

```diff
@@ -1007,3 +1008,117 @@ sources:
           - name: type
             description: '{{ doc("type") }}'
 
+      - name: sys_user_role
```

Comment:
Don't forget to add the config to disable the sources for users if they are setting the respective variable to `false`. See how we did this [here](https://github.com/fivetran/dbt_zendesk_source/blob/fceed335f9cf146781a05cd6b01557f276d54772/models/src_zendesk.yml#L245-L246).

Reply:
Please apply the same update to the other sources that may be enabled/disabled.

Reply:
thanks for the reminder!